src/modules/song/thunks.ts

```diff
@@ -74,13 +74,12 @@ export const uploadSong = createAsyncThunk(
       if ("error" in audioUploadUrlResp) return;
 
       const { uploadUrl } = audioUploadUrlResp.data;
-      const audioBinaryStr = await getFileBinary(body.audio);
 
       // upload audio to AWS, song audioUrl will be updated after it's transcoded
       await fetch(uploadUrl, {
         method: "PUT",
-        headers: { "Content-Type": "application/json" },
-        body: JSON.stringify({ file: audioBinaryStr }),
+        headers: { "Content-Type": "application/octet-stream" },
```

Comment:
devops suggested to use this so that our flac and wav files get uploaded and encoded successfully.